lib/main.coffee

```diff
@@ -46,6 +46,16 @@ module.exports =
       lintOnFly: true
       lint: (textEditor) =>
         filePath = textEditor.getPath()
+        # Check if the project directory contains .coafile
+        check = helpers.exec(@executable,
+                             ['--find-config'],
+                             {cwd: path.dirname(filePath)}).then (result) ->
+          atom.notifications.addSuccess('Check completed').catch (error) ->
+            atom.notifications.addInfo \
+              'coala: Missing Dependencies!',
+              'detail': 'Project directory does not contain .coafile.\n
+                         Please provide a .coafile for using coala.\n
+                         Make sure coala-bears meet all dependencies.'
```

Comment:
Are you sure this works as intended? When I test with removing the `.coafile` and/or `coala-bears`, the only error that's raised is from the base `linter`, which flags: `See Console for more info. (Open View -> Developer -> Toogle Developer Tools)`